content/blog/analysis-with-RCrunch.Rmd

```diff
@@ -0,0 +1,300 @@
+---
+title: "Analysis with Crunch's R Package"
+description: "Using Crunch to speed up common analysis steps"
+tags: []
+categories: ["general"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: markdown_github
+---
+
+```{r set-options, echo=FALSE}
+options(width = 110)
+```
+
+Crunch provides a collaborative platform for data analytics. Our app allows for simple, visual exploration and analysis, but we know that data scientists and developers want to harness the power of R, Python, and other languages to manipulate, cleanse, and analyze data. Crunch was designed to make collaboration easy and this doesn't stop at the transition between Crunch servers and languages like R and Python. Crunch provides an [R package](https://github.com/crunch-io/rcrunch/) that makes repetitive tasks easy to automate, in a way that feels like an integrated part of R. 
+
+Crunch's R package makes many common cleaning and analysis operations easy. Best of all, it minimizes the amount of data we need to transfer back and forth between Crunch and our local R session. This not only speeds up analysis, but updates data in Crunch instantly so collaborators can see the changes and work with the most up-to-date data as possible.
```

Comment:
@nealrichardson suggested I make the bot do a sed replace of straight apostrophes with curly ones, and I predictably groused that `pandoc --smart` would be better and less brittle. at any rate I see lots of straight apostrophes here and they bug me